src/status_im/ui/screens/chat/input/animations/expandable.cljs

```diff
@@ -9,21 +9,24 @@
 
 (def top-offset 100)
 
-(defn expandable-view-on-update [anim-value animation-height]
-  (when animation-height
-    (animation/start
-     (animation/spring anim-value {:toValue  animation-height
-                                   :friction 10
-                                   :tension  60}))))
+(defn expandable-view-on-update [anim-value]
+  (animation/start
+   (animation/spring anim-value {:toValue         -53
+                                 :friction        10
+                                 :tension         60
+                                 :useNativeDriver true})))
 
-(defview expandable-view [{:keys [key]} & elements]
-  (letsubs [anim-value         (animation/create-value 0)
+(defview expandable-view [_ & elements]
+  (letsubs [anim-value         (animation/create-value 104)
```

Comment:
Can we infer or comment those magic numbers?

Reply:
added a comment